src/view/configuration/basicSectionBridge.js

```diff
@@ -82,7 +82,8 @@ export const bridge = {
       orgId: string().required("Please specify an IMS organization ID."),
       edgeDomain: string().required("Please specify an edge domain."),
     })
-    .test("uniqueName", (instance, testContext) => {
+    // eslint-disable-next-line func-names
+    .test("uniqueName", function (instance, testContext) {
```

Comment:
Using an arrow function, you can remove the eslint comment.

```suggestion
    .test("uniqueName", (instance, testContext) => {
```